setup.py

```diff
@@ -11,8 +11,7 @@
 requirements = [
     "cached-property>=1.5; python_version < '3.8'",
     "eip712_structs",
-    "packaging",
```

Comment:
So this was not required?

Reply:
It should be included in any django distrution, but we are using it, I will put it back

Reply:
Done